### ISSUE TITLE
Home Page Widgets

### ISSUE THREAD

**Issue by Nuhser** [Nuhser/PfaKSys#25]
### Description

Currently the home page is pretty much empty. It would be nice to have some "widgets" the users can customize and reorder as they see fit.

### Possible Solution

- implement multiple widgets to let the users choose from
- add a sub-menu or something similar for the users to customize their selections and widget order
- save the configuration to the user settings in the db

### Widget Ideas

- Recent Material Changes
- Upcoming Events
- Mini-Calendar

----

### Additional Context

**Blocked by:**
- #5 
- #6
